02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,57 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>About Staci</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-
+  <header id="name">Staci Holmes-Segarra</header>
+    <p style= "font-size:30px; text-align: center">San Antonio, Texas</p>
+  <div>
+    <img src="https://scontent-lax3-1.xx.fbcdn.net/v/t1.0-1/p160x160/10357598_1647238348855062_2668382716167648205_n.jpg?oh=1ec04676b5898321d00265b8e4398f66&oe=58200377">
```

Comment:
You can center the image using 
`
img {
   display: block;
   margin: auto;
}
`